app/views/manageAgent/areYouSureMultipleYears.scala.html

```diff
@@ -19,16 +19,27 @@
 @this(govukErrorSummary : GovukErrorSummary, govukDetails : GovukDetails, govukRadios : GovukRadios, govukButton : GovukButton, mainLayout: views.html.mainLayout, formWithCSRF: FormWithCSRF,
     govukInsetText: GovukInsetText)
 
-@(agentName: String, agentCode: Long, backLink: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+@(agentName: String, agentCode: Long, backLink: String, listYears: Option[Seq[String]])(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
+
+@ratingList(tag: String) = @{
+    listYears match {
+        case Some(years) => years.sorted(Ordering.String.reverse) match {
+                            case Seq(a, b, c) => if(tag == "p") messages("ratingsListConfirmation.multiple.forAllYour.three", a, b, c) else messages("ratingsListConfirmation.multiple.heading.three", agentName, a, b, c)
+                            case Seq(a, b)    => if(tag == "p") messages("ratingsListConfirmation.multiple.forAllYour.two", a, b) else messages("ratingsListConfirmation.multiple.heading.two", agentName, a, b)
+                            case _          => ""
+                        }
+        case _ => "year not found"
```

Comment:
Can we handle the option in the controller so that we pass in listYears as a Seq[String] rather than Option[Seq[String]] to avoid potentially having "year not found" displayed to a user?